syne_tune/tuner.py

```diff
@@ -683,3 +684,30 @@ def _default_callback():
         :return: Default callback to store results
         """
         return StoreResultsCallback()
+
+    def best_config(
+        self, metric: Optional[Union[str, int]] = 0
+    ) -> Tuple[int, Dict[str, Any]]:
+        """
+        :param metric: Indicates which metric to use, can be the index or a name of the metric.
+            default to 0 - first metric defined in the Scheduler
+        :return: the best configuration found while tuning for the metric given and the associated trial-id
+        """
+        metric_name, metric_mode = metric_name_mode(
+            metric_names=self.scheduler.metric_names(),
+            metric_mode=self.scheduler.metric_mode(),
+            metric=metric,
+        )
+        trial_id, best_metric = print_best_metric_found(
+            self.tuning_status, metric_names=[metric_name], mode=metric_mode
+        )
+        config = self.trial_backend._trial_dict[trial_id].config
+
+        logger.info(
+            f"If you want to retrain the best configuration found, you can run: \n"
+            f"```tuner.trial_backend.start_trial(config={config})``` to start training from scratch\n"
+            f"or\n"
+            f"```tuner.trial_backend.start_trial(config={config}, checkpoint_trial_id={trial_id})``` to start from "
+            f"last checkpoint (your script should have stored a checkpoint)"
```

Comment:
Here or in the FAQ entry, would it make sense to explain when you would use `best_config()` versus a checkpoint?

Reply:
It is not really versus, you can only restart from a checkpoint if your script supports checkpointing which may not be the case.

I do not think it would make sense to explain checkpointing there as it has its own set of FAQ items, for instance:
* https://syne-tune.readthedocs.io/en/latest/faq.html#how-can-i-enable-trial-checkpointing